src/backend/gdb_expansion.ts

```diff
@@ -229,7 +229,13 @@ export function expandValue(variableCreate: (arg: VariableObject | string, optio
 		} else if (typeof val == "string" && val.startsWith("<...>")) {
 			ref = variableCreate(getNamespace(name));
 			val = "...";
+		} else {
+			let evaluateName = getNamespace(name);
+			if(typeof evaluateName == "string" && evaluateName.includes("static ")) {
+				evaluateName = evaluateName.replace("static ", "");
+			}
```

Comment:
You create a local variable here and adjust but never use it - or what did I miss?

Reply:
Thanks for the reminder, this piece of code is redundant.